python/mhs/import_scenarios.py

```diff
@@ -163,10 +163,23 @@ def _import_event_set(cursor, es):
         es.time_end,
         es.time_duration,
         es.description,
-        es.bibliography
+        es.bibliography,
+        es.is_prob
     ])
     return cursor.fetchone()[0]
 
+_FP_DATA_INJECT_QUERY = """
+INSERT INTO hazard.footprint_data
+    (footprint_id,the_geom,intensity)
+    (SELECT %s AS footprint_id, %s)
+"""
+
+
+def _import_footprint_data_via_query(cursor, fpid, data_query, fp):
+    query = _FP_DATA_INJECT_QUERY % (fpid, data_query)
+    verbose_message("Query = {}".format(query))
```

Comment:
This is the part I specifically would like feedback on.  The use of standard string formatting to produce a SQL query would allow a malicious user to perform UPDATE/DELETE/DROP operations as part of the SELECT.  On the other hand, the user has write access to the database and code anyway...
@micheles @daniviga @nastasi-oq

Reply:
Honestly, I do not see any security issue here, because the JSON inputs are trusted, i.e. prepared by you, and inserted only by somebody that has all the permissions.

Reply:
I agree with Michele: sources are trusted and this is not a public service, just a bunch of scripts...